security_considerations.md

```diff
@@ -10,82 +10,106 @@ if an OEM or supplier deems it necessary. In this section, we discuss several of
+Second, they SHOULD decrease the expiration timestamps on all Targets metadata uploaded to the Image repository so they expire more quickly. This can prevent attackers who compromise the Director repository from being able to choose these obsolete images. Unfortunately, this method also has a downside. These Targets metadata will need to be updated more frequently as well. To prevent accidental freeze attacks, an ECU needs to be able to update both the time from the Time Server, and metadata from the Image repository.  In this unlikely event that the ECU is able to update metadata, but not the time, it can continue working with the previously installed image, but would be unable to update to the latest image. The Director repository can detect this unlikely event using the vehicle version manifest. In this case, the OEM MAY require the owner of the vehicle to diagnose the problem at the nearest dealership or authorized mechanic.
 
 ## Broadcasting vs. unicasting metadata inside the vehicle
-An implementation of Uptane MAY have a primary unicast metadata to secondaries, which means that the primary would send metadata separately to each secondary. However, this has a downside: due to network disruptions, different ECUs may end up seeing different versions of metadata released by repositories at different times.
+An implementation of Uptane MAY have a Primary unicast metadata to Secondaries. In this scenario, the Primary would send metadata separately to each Secondary. However, this has a downside. Network disruptions can cause ECUs to see different versions of metadata released by repositories at different times.
+
+In order to mitigate this problem, it is RECOMMENDED that a Primary use a broadcast network such as CAN, CAN FD, or Ethernet to transmit metadata to all of its Secondaries at the same time. Primaries MAY use the ECUModule.MetadataFiles message to do so. Note that this still does not guarantee that ECUs will not end up seeing different versions of metadata released by repositories at different times. This is because network traffic between Primaries and Secondaries may still get disrupted, especially if they are connected through intermediaries, such as gateways. Nevertheless, it should still be better than unicasting.
```

Comment:
```suggestion
In order to mitigate this problem, it is RECOMMENDED that a Primary use a broadcast network such as CAN, CAN FD, or Ethernet to transmit metadata to all of its Secondaries at the same time. Note that this still does not guarantee that ECUs will not end up seeing different versions of metadata released by repositories at different times. This is because network traffic between Primaries and Secondaries may still get disrupted, especially if they are connected through intermediaries, such as gateways. Nevertheless, it should still be better than unicasting.
```